### ISSUE TITLE
Special keys are not working ([insert], [ctrl], [alt], [caps])

### ISSUE THREAD

**Issue by oluckyman** [bitovi/syn#18]
``` javascript
Syn.click({}, el).type("[caps]aaa"); // types "capsaaa". Expected: "AAA"
```

``` javascript
Syn.click({}, el).type("[ctrl]"); // types "ctrl". Expected: ""
```

``` javascript
Syn.click({}, el).type("[ctrl]")
// ...
Syn.click({}, el).type("a"); // behave like ctrl is pressed.
```

How to unpress [ctrl] after `type("[ctrl]")`?

**Comment by koalixCZ** [comment]
There is a problem in the file mouse.support.js - missing checkSupport function.

```javascript
var syn = require('./synthetic');
require('./mouse');

if (!document.body) {
	syn.schedule(function () {
		checkSupport(syn);
	}, 1);
} else {
	window.__synthTest = function () {
		syn.support.linkHrefJS = true;
	};
```
In code is condition
```javascript
syn.support.ready === 2
```
and this never occurs, because the scheduler is not able to invoke the code in the else branche. Solution is to encapsulate this code into the appropriate function:

```javascript
if (!document.body) {
    syn.schedule(function () {
        checkSupport(syn);
    }, 1);
} else {
    checkSupport(syn);
}
function checkSupport(syn) {
    window.__synthTest = function () {
        syn.support.linkHrefJS = true;
    };
```

This seems works for me. I will try to fix it.

**Comment by kdillon** [comment]
I am working on this issue.

**Comment by kdillon** [comment]
This will be resolved in 0.4.3 the fix is already submitted
